package.json

```diff
@@ -47,7 +49,7 @@
   },
   "devDependencies": {
     "coveralls": "^3.0.4",
-    "eslint": "^6.0.0",
+    "eslint": "^5.3.0",
```

Comment:
Nice work @swisskid95, Is there any particular reason you chose to update the existing version of eslint to a lower one?

Reply:
Yeah, I got an error that the version of the config-airbnb-base is not peer-dependency with the eslint version originally specified. it suggested a downgrade to this particular version.

Reply:
Oh okay. cool